dbms/src/Functions/tests/gtest_cast_json_as_string.cpp

```diff
@@ -0,0 +1,354 @@
+        0x3,
+        0x5b,
+        0x0,
+        0x0,
+        0x0,
+        0x2,
+        0x0,
+        0x0,
+        0x0,
+        0x22,
+        0x0,
+        0x0,
+        0x0,
+        0x9,
+        0x12,
+        0x0,
+        0x0,
+        0x0,
+        0x9,
+        0x1a,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x1,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x2,
+        0x0,
+        0x0,
+        0x0,
+        0x22,
+        0x0,
+        0x0,
+        0x0,
+        0x9,
+        0x12,
+        0x0,
+        0x0,
+        0x0,
+        0x9,
+        0x1a,
+        0x0,
+        0x0,
+        0x0,
+        0x2,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x3,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x2,
+        0x0,
+        0x0,
+        0x0,
+        0x3c,
+        0x0,
+        0x0,
+        0x0,
+        0x9,
+        0x12,
+        0x0,
+        0x0,
+        0x0,
+        0x3,
+        0x1a,
+        0x0,
+        0x0,
+        0x0,
+        0x4,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x2,
+        0x0,
+        0x0,
+        0x0,
+        0x22,
+        0x0,
+        0x0,
+        0x0,
+        0x9,
+        0x12,
+        0x0,
+        0x0,
+        0x0,
+        0x9,
+        0x1a,
+        0x0,
+        0x0,
+        0x0,
+        0x5,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x6,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0,
+        0x0};
+    str_col->insertData(reinterpret_cast<const char *>(bj2), sizeof(bj2) / sizeof(UInt8));
+    str_col->insertData("", 0);
+    str_col->insertData(reinterpret_cast<const char *>(bj9), sizeof(bj9) / sizeof(UInt8));
+    ColumnUInt8::MutablePtr col_null_map = ColumnUInt8::create(3, 0);
+    ColumnUInt8::Container & vec_null_map = col_null_map->getData();
+    vec_null_map[1] = 0;
+    auto json_col = ColumnNullable::create(std::move(str_col), std::move(col_null_map));
+    auto input_col = ColumnWithTypeAndName(std::move(json_col), nullable_string_type_ptr, "input0");
+
+    auto output_col = createColumn<Nullable<String>>({"[{\"a\": 1, \"b\": true}, 3, 3.5, \"hello, world\", null, true]", {}, "[[0, 1], [2, 3], [4, [5, 6]]]"});
+    auto res = executeFunction(func_name, input_col);
+    ASSERT_COLUMN_EQ(res, output_col);
+
+    /// ColumnConst(null)
+    auto null_input_col = createConstColumn<Nullable<String>>(3, {});
+    output_col = createConstColumn<Nullable<String>>(3, {});
+    res = executeFunction(func_name, null_input_col);
+    ASSERT_COLUMN_EQ(res, output_col);
+
+    /// ColumnVector(non-null)
```

Comment:
add test for ColumnVector(null)?

Reply:
Do you mean columnvector nullable, and some element is null? If so, normal case include this.

Reply:
> 

I mean ColumnVector(nullable) but only have null values.

Reply:
get it, I'll add it.

Reply:
Done